@uportal/content-carousel/src/components/ContentCarousel.html

```diff
@@ -8,19 +8,32 @@
     </slot>
   </template>
   <template v-else>
-    <slick ref="slick" :options="computedSlickOptions">
-      <template v-for="item in computedItems">
-        <slot name="slide" :item="item">
-          <div class="slick-item">
-            <p v-if="!item.imageUrl" v-html="item.description" class="slick-item-description"></p>
-            <a :href="item.destinationUrl" class="slick-title">
-              <img v-if="item.imageUrl" :src="item.imageUrl" :alt="item.altText" class="slick-image">
-              {{ item.title }}
-            </a>
-          </div>
-        </slot>
-      </template>
-      <slot v-if="type === 'passthrough'" />
-    </slick>
+    <div class="slick-inner">
+      <slick ref="slick" :options="computedSlickOptions">
+        <template v-for="item in computedItems">
+          <slot name="slide" :item="item">
+            <div v-if="computedSlickOptions.slidesToShow === 1" class="slick-hero-container">
+              <a :href="item.destinationUrl">
+                <img v-if="item.imageUrl" :src="item.imageUrl" :alt="item.altText">
+                <template v-if="!item.imageUrl">{{ item.title }}</template>
+                <div v-if="item.altText" class="slick-hero-text">
+                  <p style="margin: 0; padding: 10px;" v-bind:style="alignText">{{ item.altText }}</p>
```

Comment:
any chance that inline styles could be moved to sass?

Reply:
oops copypasta.